idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
The choice key matches what is used in fan coil. I don't mind changing this now since I have a few other details to correct. The term is meant to describe the ASHRAE 90.1 method of two- or variable-speed fans used with DX and water coils, and allowing low speed fan during low load conditions. If it were to change, I do like SingleZoneVAV although UnitarySystem has a multi-zone capability.

    ZoneHVAC:FourPipeFanCoil,
     A3 , \field Capacity Control Method
        \required-field
        \type choice
        \key ConstantFanVariableFlow
        \key CyclingFan
        \key VariableFanVariableFlow
        \key VariableFanConstantFlow
        \key MultiSpeedFan
        \key ASHRAE90VariableFan